fe/fe-core/src/main/java/com/starrocks/clone/TabletScheduler.java

```diff
@@ -296,6 +299,10 @@ public synchronized AddResult addTablet(TabletSchedCtx tablet, boolean force) {
         return AddResult.ADDED;
     }
 
+    public synchronized void forceCleanSchedQ() {
```

Comment:
Is it necessary to use synchronized?

Reply:
no, acctually shouldn't use, synchronized is used inside, i will remove it , good catch 👍🏻